tasks/express/easy/app.js

```diff
@@ -1,10 +1,16 @@
 // ExpressJS - Easy
 
-const { get } = require("cheerio/lib/api/traversing");
 const express = require("express");
 const app = express();
 const port = 6969;
 
+app.get("/", (req, res) => {
+    res.send("Hello, Express!");
+});
+app.listen(port, () => {
+    console.log("Server is running on port 6969");
```

Comment:
Looks good!

The port number is already defined as variable so you could also use that like this:
```bash
console.log(`Server is running on port ${port}`);
```